### ISSUE TITLE
Tag: Accessibility - "Tag" is read out when interactive=false

### ISSUE THREAD

**Issue by summer2200** [UI5/webcomponents#12475]
### Describe the bug

When a non-interactive Tag is read out by screen reader, it's announcing "Tag [tag content]". Expected behavior should be just announcing the tag content without "Tag".

### Isolated Example

https://stackblitz.com/edit/github-g7wrb9e1?file=src%2FApp.tsx

### Reproduction steps

1. Press Tab key to focus on the first timeline item, the tag was read out as "TagSingle Role"

### Expected Behaviour

Should just read out the tag content without "Tag".

### UI5 Web Components for React Version

2.14.0

### UI5 Web Components Version

2.14.0

### Browser

Chrome

### Operating System

MacOS

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.

**Comment by hinzzx** [comment]
Hi @UI5/ui5-team-ctr-rod , could you please check the issue with the `ui5-tag` component ?

The reporter expects the content of non-interactive `ui5-tag` component to be read out, without starting with "TAG" before.

**Comment by GerganaKremenska** [comment]
Hello @summer2200,

UI5 Web Components provide the prerequisites for screen reader support based on the HTML, ARIA, and WCAG standards. All screen readers that follow those standards should work fine. Nevertheless, there are deviations in the interpretation depending on the combination of browser and screen reader. UI5 Web Components focus on compliance with the standards, rely on automated checks for accessibility and manual tests with reference testing environments. 

For Screen Reader Support, we recommend using JAWS 2025 + Chrome (latest), and for HTML/ARIA validation the recommended testing tool is [Access Assistant](https://chrome.google.com/webstore/detail/access-assistant/ojiighldhdmahfdnhfdebnpmlbiemdfm). UI5 Web Components support other environments to the extent of providing a valid HTML and ARIA implementation following the WCAG standards.

When reporting issues with different testing environments, please ensure that the issue is not false positive, a real accessibility concern is present and there is an impact over the users. Therefore, we recommend to retest the issue using the mentioned reference testing tools and additionally with plain HTML.

In order to process the issue, we would like to have the following information provided:

Issue description:

- Reference to the suspected violated accessibility requirement (e.g. Web Content Accessibility Guidelines, WCAG 2.2, BITV 2.0, EN 301 549):
- Screen Reader:


NOTE: An non interactive tag can not be focused, it can only be read in virtual cursor mode. But the output is the same. 

If a violated accessibility requirement can not be found, please close the issue. 

Best Regards,
Gergana

**Comment by summer2200** [comment]
Hi @GerganaKremenska ,

This is actually an issue raised by one of our customer, they're using NVDA for a11y testing. But currently a violated accessibility requirement cannot be found, so I'm closing this issue for now.

Thank you for your reply!

Thanks,
Summer

**Comment by mastanshaikk** [comment]
> Hello [@summer2200](https://github.com/summer2200),
> 
> UI5 Web Components provide the prerequisites for screen reader support based on the HTML, ARIA, and WCAG standards. All screen readers that follow those standards should work fine. Nevertheless, there are deviations in the interpretation depending on the combination of browser and screen reader. UI5 Web Components focus on compliance with the standards, rely on automated checks for accessibility and manual tests with reference testing environments.
> 
> For Screen Reader Support, we recommend using JAWS 2025 + Chrome (latest), and for HTML/ARIA validation the recommended testing tool is [Access Assistant](https://chrome.google.com/webstore/detail/access-assistant/ojiighldhdmahfdnhfdebnpmlbiemdfm). UI5 Web Components support other environments to the extent of providing a valid HTML and ARIA implementation following the WCAG standards.
> 
> When reporting issues with different testing environments, please ensure that the issue is not false positive, a real accessibility concern is present and there is an impact over the users. Therefore, we recommend to retest the issue using the mentioned reference testing tools and additionally with plain HTML.
> 
> In order to process the issue, we would like to have the following information provided:
> 
> Issue description:
> 
> * Reference to the suspected violated accessibility requirement (e.g. Web Content Accessibility Guidelines, WCAG 2.2, BITV 2.0, EN 301 549):
> * Screen Reader:
> 
> NOTE: An non interactive tag can not be focused, it can only be read in virtual cursor mode. But the output is the same.
> 
> If a violated accessibility requirement can not be found, please close the issue.
> 
> Best Regards, Gergana

@GerganaKremenska - The response provided is too generic and does not address the specific issue raised. The concern is that the word "Tag" is being announced to users. While it is technically accurate to refer to this as a "Tag" control in a non-interactive mode. However, end users may not understand what "Tag" means in this context.

If there are approved WCAG semantics or recommended terminology for conveying "Tag" to end users—so that the control is clearly identified and understood—please share those references. This will help us provide clear guidance to the customer.

**Comment by GerganaKremenska** [comment]
Hello @mastanshaikk, 

We understand that accessibility can be interpreted in various ways, which is why we adhere to established standards to ensure our framework is accessible and compatible with screen readers.

It's important to note that individual users may have their own interpretations of how a control should function. However, in situations where there is no clear violation of WCAG guidelines, we follow our design, which specify that tags should always be announced to help distinguish these elements from basic text.

Our current practice does not involve distributing specific documentation pertaining to WCAG semantics or recommended terminology for conveying "Tag" to end users.

Our approach is centered around adherence to established accessibility standards to ensure consistency and understanding across our framework. We encourage users to refer directly to the WCAG guidelines, which contain extensive resources and information regarding semantic practices and terminology that may assist in providing clear guidance to customers.
Should you identify a particular guideline that you believe is applicable and would like to discuss further, please feel free to share it with us and reopen the ticket

Best Regards,
Gergana
